src/realm/obj.hpp

```diff
@@ -276,6 +278,7 @@ class Obj {
     Dictionary get_dictionary(StringData col_name) const;
 
     CollectionBasePtr get_collection_ptr(ColKey col_key) const;
+    LinkCollectionPtr get_linkcollection_ptr(ColKey col_key) const;
```

Comment:
We have a lot of these kinds of methods, and it's getting confusing. But in this case, it's probably worth it. :-)